usaspending_api/awards/models.py

```diff
@@ -102,9 +103,9 @@ class Award(DataSourceTrackedModel):
         ('L', 'Loan'),
     )
 
-    type = models.CharField(max_length=5, choices=AWARD_TYPES, verbose_name="Award Type")
+    type = models.CharField(max_length=5, choices=AWARD_TYPES, verbose_name="Award Type", null=True)
```

Comment:
@bsweger I agree that we may want to create an "unknown" type instead of leaving it null.